css/main.css

```diff
@@ -109,3 +109,11 @@ button:hover {
   -ms-flex-align: center !important;
   align-items: center !important;
```

Comment:
You should not use !important as vikram told. I don't exactly remember why but it should not be used.

Reply:
Ack. I'll try to minimize the usage of !important. in following PRs